packages/app/cypress/e2e/cypress-in-cypress-component.cy.ts

```diff
@@ -1,7 +1,7 @@
 import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
 import { snapshotAUTPanel } from './support/snapshot-aut-panel'
 
-describe('Cypress In Cypress', { viewportWidth: 1500 }, () => {
+describe('Cypress In Cypress CT', { viewportWidth: 1500, defaultCommandTimeout: 10000 }, () => {
```

Comment:
This extra long timeout here is useful as we have some assertions that require the test in the internal cypress app to complete, and have certain content in the reporter. An otherwise passing test currently can still be flaky if the internal test doesn't complete in time.

Reply:
I did add a helper to validate expected output for inner Cypress test runs, but it's pretty well scoped to the runner specs at this point: https://github.com/cypress-io/cypress/blob/10.0-release/packages/app/cypress/e2e/runner/support/spec-loader.ts#L1-L13. 

This is just a heads up. You could extract that out to target those specific assertions more granularly, but I don't think that's much of a gain over what you have here.